toolchains/python/python.bzl

```diff
@@ -36,20 +44,25 @@ toolchain(
     target_compatible_with = {target_constraints},
 )
 """.format(
-        python2_runtime = label_string(repository_ctx.attr.python2_runtime),
-        python3_runtime = label_string(repository_ctx.attr.python3_runtime),
+        python2_runtime = label_string(python2_runtime),
+        python3_runtime = label_string(python3_runtime),
         exec_constraints = exec_constraints,
         target_constraints = target_constraints,
     ))
 
+    repository_ctx.file("defs.bzl", executable = False, content = """
+# Generated by rules_nixpkgs/toolchains/python/python.bzl
+interpreter = "{}"
+    """.format(repository_ctx.attr.python3_repo + ":bin/python"))
+
```

Comment:
The case where `repository_ctx.attr.python3_repo` is empty doesn't seem to be handled here. In that case it should probably fall back to the python2 one.

Reply:
Yeah, good point. Will do this now.